transformers/events/dent/repository.go

```diff
@@ -17,19 +17,18 @@
 package dent
 
 import (
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
 	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
-
-	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
 )
 
-type DentRepository struct {
+type Repository struct {
```

Comment:
Why are we keeping a transformer specific repository at all? Seems like we could just initialize the shared `factories/transformer` with a database connection and pass that directly to `event.Create`. Then we could remove the `Repository` from the initializer entirely, right?

Reply:
yep.

Reply:
Available for whoever wants to shepherd the dual PRs :) We just decided to do it in steps a while back, moving stuff over, then removing the repos.